### PR TITLE
Add watchdog commands

### PR DESCRIPTION
**Pull request by StykMartin**
Add all watchdog commands that are currently supported.

Signed-off-by: Martin Styk <mart.styk@gmail.com>

### CODE REVIEW

File: Client/src/bkr/future/param_types.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Copyright Contributors to the Beaker project.
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+from typing import List
```
Comment (JohnVillalovos) [inline Client/src/bkr/future/param_types.py:4]: Type-hints. I like!

File: Client/src/bkr/future/watchdog/list_watchdog.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+@click.command("list")
+@click.argument(
+    "task_spec",
+    nargs=-1,
+    type=TaskSpecParamType([TaskSpecType.T]),
+)
+@pass_api
+def list_watchdog(api, task_spec: List[TaskSpec]):
+    """Extends the watchdog time."""
```
Comment (JohnVillalovos) [inline Client/src/bkr/future/watchdog/list_watchdog.py:64]: Doc-string seems incorrect.

File: Client/src/bkr/future/watchdog/extend.py [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+Synopsis
+--------
+
+|  :program:`bkr watchdog extend` [*options*] <taskspec>
+|       [:option:`--by` <seconds>]
+
+Description
+-----------
+
+Extends the watchdog time for one or more recipes by specifying one
```
Comment (JohnVillalovos) [inline Client/src/bkr/future/watchdog/extend.py:21]: Does it "extend" the watchdog time? If the current watchdog is set for 30 minutes from now, if I "extend" it for 10 minutes is the new watchdog 40 minutes or 10 minutes?

File: Client/src/bkr/future/watchdog/list_watchdog.py (hunk 2 above)
Comment (StykMartin) [reply Client/src/bkr/future/watchdog/list_watchdog.py:64]: Corrected.

**Review by JohnVillalovos — APPROVED** [review]

**Pull request merged**